src/Storages/MergeTree/MergeTreeSettings.cpp

```diff
@@ -286,6 +286,15 @@ namespace ErrorCodes
     - `single_stream` — Use the standard serialization format with inline sizes.
     - `with_size_stream` — Use a separate size stream for top-level `String` columns.
     )", 0) \
+    DECLARE(MergeTreeNullableSerializationVersion, nullable_serialization_version, "allow_sparse", R"(
```

Comment:
Let's set basic serialization as the default in the first release with sparse nullable types, to allow downgrades from the new version.

Reply:
Sure.